src/main/java/com/jw/service/REVIEW-VINI.MD

```diff
@@ -0,0 +1,2 @@
+Few comments:
+- This package contains services, repositories and mappers, please split it accordingly. Same for Queue*, I would add it into a new package called infrastructure(but these classes are minor, it can(maybe) fit in here) 
```

Comment:
I created and moved accordingly Repository and Mapper classes, but tbh I feel like QueueReader and QueueWriter are much more than connectivity with rabbitmq, and taking into consideration logic they perform I think they fit in a service package. I'll leave them for now, but open to discuss